bigtable-dataflow-parent/bigtable-dataflow-import/README.md

```diff
@@ -36,7 +36,7 @@ java -jar bigtable-dataflow-import-0.9.5.1-shaded.jar import \
     --bigtableProjectId=[your_project_id] \
     --bigtableInstanceId=[your_instance_id] \
     --bigtableTableId=[your_table_id] \
-    --destination=gs://[bucket_name]/[export_directory] \
+    '--filePattern=gs://[bucket_name]/[import_directory]/*' \
```

Comment:
You probably just want to quote the RHS of the flag, not the flag itself:

    --filePattern='gs://[bucket_name]/[import_directory]/*' \